doc/user/content/release-notes.md

```diff
@@ -103,7 +103,7 @@ List new features before bug fixes.
 
 {{% version-header v0.12.0 %}}
 
-- No changes yet.
+- Added the `pg_type_is_visible` [compatibility function](/sql/functions#postgresql-compatibility-func).
```

Comment:
"Add the ...". A release note should have an implicit "This release will ..." at the start.